Resources/migration.yml

```diff
@@ -129,7 +129,7 @@ HyperlinkBookSpaceLaw: BookSecurity
 HyperlinkBookSupernanny: BookHowToSurvive
 SpawnPointCataloguer: SpawnPointLibrarian
 SpawnPointCyborg: SpawnPointBorg
-SpawnPointMedicalCyborg: SpawnPointBorgMedical
+SpawnPointMedicalCyborg: SpawnPointBorg
```

Comment:
This could probably just be null since standard borgs are also mapped elsewhere on maps.